packages/langium/test/dependency-injection.test.ts

```diff
@@ -59,7 +59,7 @@ describe('A non-cyclic dependency', () => {
     });
 
     test('should be result on an idempotent api call', () => {
-        const api = inject({ dep: () => ({ }) });
+        const api = inject({ dep: () => ({}) });
         expect(api.dep).toBe(api.dep);
```

Comment:
This is a tautology, isn't it? Does that make any sense?

Reply:
I think it is meant as: two subsequent calls, return the same result.

Reply:
At the first glance I expected a test expression like
```
expect(api.dep()).toBe(api.dep());
```
... i.e. `api.dep()` instead of `api.dep`.

I overlooked the fact that `api` is the proxy object created by `inject(...)`, and that the supplier function `dep` is called internally once the property `dep` is requested ... what this PR is more or less about ... my bad!